packages/mux-audio-react/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "composite": true,
     "strict": true,
     "noImplicitAny": true,
-    "moduleResolution": "Node",
+    "moduleResolution": "Bundler",
```

Comment:
Updating `moduleResolution` to `"Bundler"` for all elements packages for more accurate description of how we consume modules in the monorepo. Resolves issues with player.style not publishing CJS modules. For more, see:

* https://www.typescriptlang.org/tsconfig/#moduleResolution
* https://www.typescriptlang.org/docs/handbook/modules/theory.html#module-resolution

**NOTE:** In one of the above links, there is mention that "[`moduleResolution` of type `"Bundler"`] requires setting `module` to `esnext`." Since we only use `tsc` to generate our TypeScript declaration files (aka `.d.ts` files), this should not be relevant for us afaik.